diff --git a/node_modules/solid-js/types/jsx.d.ts b/node_modules/solid-js/types/jsx.d.ts
index 10de29f..e8d959b 100644
--- a/node_modules/solid-js/types/jsx.d.ts
+++ b/node_modules/solid-js/types/jsx.d.ts
@@ -1211,7 +1211,7 @@ export namespace JSX {
     autocorrect?: "on" | "off" | undefined;
     contenteditable?: "true" | "false" | boolean | "plaintext-only" | "inherit" | undefined;
     dir?: HTMLDir | undefined;
-    draggable?: boolean | "false" | "true" | undefined;
+    draggable?: "false" | "true" | undefined;
     enterkeyhint?: "enter" | "done" | "go" | "next" | "previous" | "search" | "send" | undefined;
     exportparts?: string | undefined;
     hidden?: boolean | "hidden" | "until-found" | undefined;
