diff --git a/node_modules/solid-js/types/jsx.d.ts b/node_modules/solid-js/types/jsx.d.ts
index 3b93d3e..bd83e39 100644
--- a/node_modules/solid-js/types/jsx.d.ts
+++ b/node_modules/solid-js/types/jsx.d.ts
@@ -909,7 +909,7 @@ export namespace JSX {
     contenteditable?: boolean | "plaintext-only" | "inherit" | undefined;
     contextmenu?: string | undefined;
     dir?: HTMLDir | undefined;
-    draggable?: boolean | "false" | "true" | undefined;
+    draggable?: "false" | "true" | undefined;
     hidden?: boolean | "hidden" | "until-found" | undefined;
     id?: string | undefined;
     is?: string | undefined;
